src/seedu/addressbook/AddressBook.java

```diff
@@ -420,7 +420,7 @@ private static String getMessageForSuccessfulAddPerson(String[] addedPerson) {
      * @return feedback display message for the operation result
      */
```

Comment:
Always check the method comments for changes, after you modify the method's behaviour.